library not compatible with Java 7

On a consuming project, iust got this for the latest SNAPSHOT:
```
Download https://oss.jfrog.org/oss-snapshot-local/org/m2ci/msp/artk/0.5-SNAPSHOT/artk-0.5-20161110.095817-7.jar
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileGroovy'.
> org/m2ci/msp/ema/est/ESTParser : Unsupported major.minor version 52.0
```

--- a/build.gradle
+++ b/build.gradle
@@ -8,6 +8,9 @@ group 'org.m2ci.msp'
 version '0.5-SNAPSHOT'
 description 'Articulatory data ToolKit'
 
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
 repositories {
     jcenter()
 }

